[2024-07-30T10:27:10.494+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:27:10.541+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-11T00:00:00+00:00 [queued]>
[2024-07-30T10:27:10.556+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-11T00:00:00+00:00 [queued]>
[2024-07-30T10:27:10.557+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:27:10.656+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-11 00:00:00+00:00
[2024-07-30T10:27:10.664+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1082) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:27:10.666+0000] {standard_task_runner.py:63} INFO - Started process 1086 to run task
[2024-07-30T10:27:10.665+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-11T00:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpssd3w6yk']
[2024-07-30T10:27:10.667+0000] {standard_task_runner.py:91} INFO - Job 155: Subtask Run_data_quality_checks
[2024-07-30T10:27:10.801+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-11T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:27:11.066+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-11T00:00:00+00:00'
[2024-07-30T10:27:11.069+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:27:11.106+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:27:11.136+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:11.168+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:27:11.175+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:11.179+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 1833 records
[2024-07-30T10:27:11.182+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:27:11.203+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:11.229+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:27:11.382+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:11.385+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 78 records
[2024-07-30T10:27:11.386+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:27:11.403+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:11.531+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:27:11.537+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:11.541+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:27:11.542+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:27:11.563+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:11.652+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:27:11.696+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:11.709+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:27:11.711+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:27:11.749+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:11.803+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:27:11.810+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:11.813+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 1833 records
[2024-07-30T10:27:11.815+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:27:11.851+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-11T00:00:00+00:00, execution_date=20181111T000000, start_date=20240730T102710, end_date=20240730T102711
[2024-07-30T10:27:11.967+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:27:12.052+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:27:12.085+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
